Lectures/code_review_example.R

```diff
@@ -0,0 +1,32 @@
+library(palmerpenguins)
+library(ggplot2)
+
+(penguins
+	%>% filter(island == "Torgersen")
+	%>% group_by(year, species) 
+	%>% summarise(count = n()) 
```

Comment:
dplyr functions are being used without loading the dplyr library. Add library our use dplyr:: namespace